src/Symfony/Component/Validator/Constraints/CssColor.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\InvalidArgumentException;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class CssColor extends Constraint
+{
+    public const VALIDATION_MODE_HTML5 = 'html5';
+    public const VALIDATION_MODE_LONG = 'css_long';
+    public const VALIDATION_MODE_SHORT = 'css_short';
+    public const VALIDATION_MODE_NAMED_COLORS = 'named_colors';
+    public const INVALID_FORMAT_ERROR = '454ab47b-aacf-4059-8f26-184b2dc9d48d';
+
+    protected static $errorNames = [
+        self::INVALID_FORMAT_ERROR => 'STRICT_CHECK_FAILED_ERROR',
+    ];
+
+    /**
+     * @var string[]
+     */
+    private static $validationModes = [
+        self::VALIDATION_MODE_HTML5,
+        self::VALIDATION_MODE_LONG,
+        self::VALIDATION_MODE_SHORT,
+        self::VALIDATION_MODE_NAMED_COLORS,
+    ];
+
+    public $message = 'This value is not a valid hexadecimal color.';
+    public $mode;
+    public $normalizer;
+
+    public function __construct(
+        array $options = null,
+        string $message = null,
+        string $mode = null,
+        callable $normalizer = null,
+        array $groups = null,
+        $payload = null
+    ) {
```

Comment:
should be on one line

Reply:
No, this is consistent with other constraints.